spec/changelog.yml

```diff
@@ -12,6 +12,11 @@
   - description: Add support for `slo` assets.
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/767
+- version: 3.2.3-next
+  changes:
+  - description: Update description of agent.base_image in system tests configuration files.
+    type: bugfix
+    link: https://github.com/elastic/package-spec/pull/796
```

Comment:
Maybe we don't need to add changelog for this.

Reply:
Ok, as it is just a description change, no need to do this.
Thanks!
I'll remove this changelog entry